src/elements/form-field/form-field/form-field.ts

```diff
@@ -443,6 +444,11 @@ class SbbFormFieldElement extends SbbNegativeMixin(SbbHydrationMixin(LitElement)
     }
     this.toggleAttribute('data-readonly', this._input.hasAttribute('readonly'));
     this.toggleAttribute('data-disabled', this._input.hasAttribute('disabled'));
+    if (this._input.hasAttribute('disabled')) {
```

Comment:
Maybe it's nicer to move this into the autocomplete? the trigger is there and then it works also for input elements not inside a form field.

Reply:
I agree, I added a listener for both disabled and readonly properties on the trigger